logdna/client.go

```diff
@@ -62,7 +62,37 @@ type channelResponse struct {
 	URL             string            `json:"url,omitempty"`
 }
 
-// MakeRequestView makes a request to the config-api and parses and returns the response
+// MakeRequestAlert makes a request to the config-api with alert payload data and parses and returns the response
```

Comment:
are these comments good?

Reply:
I think the comment is very helpful, but I like [well-written and judicial] comments. And if we ever want to generate APIs docs by pulling out comments, we will need them.